test/model/message_test.dart

```diff
@@ -273,75 +273,122 @@ void main() {
       });
 
       test('message topic both unresolved and edited update', () async {
-        await sendEvent(message, eg.updateMessageMoveEvent([message, otherMessage],
-          origTopic: '✔ new topic',
+        await prepareOrigMessages(origTopic: '✔ new topic');
+        await store.handleEvent(eg.updateMessageEventMoveFrom(
+          origMessages: origMessages,
           newTopic:  'new topic 2'));
+        checkNotified(count: 2);
         check(store).messages.values.every(((message) => message.editState.equals(MessageEditState.moved)));
       });
 
       test('message stream moved update', () async {
-        await sendEvent(message, eg.updateMessageMoveEvent([message, otherMessage],
-          origTopic: 'topic',
-          newTopic: 'topic',
-          newStreamId:  20));
-        check(store).messages.values.every(((message) => message.editState.equals(MessageEditState.moved)));
+        await prepareOrigMessages(origTopic: 'topic');
+        await store.handleEvent(eg.updateMessageEventMoveFrom(
+          origMessages: origMessages,
+          newTopic:  'topic-new',
+          newStreamId: 20));
+        checkNotified(count: 2);
+        check(store).messages.values.every(((message) =>
+          message.isA<StreamMessage>()
+            ..editState.equals(MessageEditState.moved)
+            ..displayRecipient.equals(null)));
+      });
+
+      test('message stream moved without topic change', () async {
+        await prepareOrigMessages(origTopic: 'topic');
+        await store.handleEvent(eg.updateMessageEventMoveFrom(
+          origMessages: origMessages,
+          newStreamId: 20));
+        checkNotified(count: 2);
+        check(store).messages.values.every(((message) =>
+          message.isA<StreamMessage>()
+            ..editState.equals(MessageEditState.moved)
+            ..displayRecipient.equals(null)));
       });
 
       test('message is both moved and updated', () async {
-        await sendEvent(message, eg.updateMessageMoveEvent([message, otherMessage],
-          origTopic: 'topic',
-          newTopic: 'topic',
-          newStreamId:  20,
-          origContent: 'old content',
+        await prepareOrigMessages(origTopic: 'topic', origContent: 'old content');
+        await store.handleEvent(eg.updateMessageEventMoveFrom(
+          origMessages: origMessages,
+          newTopic:  'new topic',
+          newStreamId: 20,
           newContent: 'new content'));
-        check(store).messages[message.id].editState.equals(MessageEditState.edited);
-        check(store).messages[otherMessage.id].editState.equals(MessageEditState.moved);
+        checkNotified(count: 2);
+        check(store).messages[origMessages[0].id].editState.equals(MessageEditState.edited);
+        check(store).messages[origMessages[1].id].editState.equals(MessageEditState.moved);
+      });
+
+      test('message stream moved without topic change', () async {
```

Comment:
Looks like there's two tests with this name. Is it accurate for both? If so it should still get some extra words to distinguish what each test is about.

Reply:
Looks like one of them is an artifact of a misapplied fixup commit. Neither is needed, so both are removed. Renaming an old test that covers this same story.